src/Core/Components/Slider/FluentSliderLabel.razor.cs

```diff
@@ -38,10 +38,16 @@ public FluentSliderLabel()
     public bool? HideMark { get; set; }
 
     /// <summary>
-    /// Gets or sets disabled state of the label. This is generally controlled by the parent.
+    /// Gets the disabled state of the label. This is controlled by the owning <see cref="FluentSlider{TValue}"/>.
     /// </summary>
     [Parameter]
-    public bool? Disabled { get; set; }
+    public bool? Disabled {
```

Comment:
That's a breaking change: previously it was possible to use `FluentSliderLabel` outside `FluentSlider` (I don't know why it was created that way, but it is :-)

So, probably you should manage the possibility to have `Owner is null` and keep an private Disabled value in this case.

```csharp
[Parameter]
public bool? Disabled
{
    get => Owner is null ? _disabled : Owner.Disabled;
    set => _disabled = Owner is null ? value : Owner.Disabled;
}
```

Reply:
I don't know why it was created like that either 🤔

Your change is indeed non-breaking so better. Will change it